app/assets/stylesheets/pages/_about.scss

```diff
@@ -0,0 +1,42 @@
+.card {
```

Comment:
These styles look good! However, Steve has already made corrections to the .card component 😄